src/ctapipe_io_nectarcam/__init__.py

```diff
@@ -38,14 +38,16 @@
     N_GAINS, N_PIXELS, N_SAMPLES
 )
 from .containers import NectarCAMDataContainer, NectarCAMServiceContainer, \
-    NectarCAMEventContainer
+    NectarCAMEventContainer, NectarCAMDataStreamContainer
 from .anyarray_dtypes import (
     CDTS_AFTER_37201_DTYPE,
     CDTS_BEFORE_37201_DTYPE,
     TIB_DTYPE,
 )
 from .version import __version__
 
+from IPython import embed
```

Comment:
Is this really needed? The only call to embed I see is in a block of code commented out?

Reply:
No, it's a remaining of some debugging process. I've removed it.
I was too hurry in the commit process in seems 😅